negotiation/contract.negotiation.protocol.md

```diff
@@ -39,6 +39,21 @@ The CN state machine is represented in the following diagram. Note that transiti
 Transitions marked with `C` indicate a message sent by the consumer, transitions marked with `P` indicate a provider message. Terminal states are final; the state machine may
 not transition to another state. A new CN may be initiated if, for instance, the CN entered the `TERMINATED` state due to a network issue.
 
+Following table gives non-normative examples of transitions:
+
+| Origin State               | Actor  | Transition Trigger Message                | Target State | Non-normative Example                                                                                                                                                                                                                                                                                                                    |
+|----------------------------|--------|-------------------------------------------|--------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| INIT                       | C      | ContractRequestMessage                    | REQUESTED    | The consumer requests a contract, without previous direct interaction with provider, e.g. through the usage of a central cataloging system, which provides all necessary information about the provider, to directly request a data contract.                                                                                            |
+| INIT                       | P      | ContractOfferMessage                      | OFFERED      | The provider sends a contract offer directly to a consumer without previous interaction, e.g. because the provider knows about desired data (e.g., consumer placed a search for data request in a catalog) or consumer/provider are in an existing business relationship and provider wants to ship information to its business partner. |
+| REQUESTED                  | P      | ContractOfferMessage                      | OFFERED      | The provider sends an offering for a requested resource.                                                                                                                                                                                                                                                                                 |
```

Comment:
Naming
```suggestion
| REQUESTED                  | P      | ContractOfferMessage                      | OFFERED      | The provider sends a (counter) contract offer for a requested dataset, based on a previous contract request.                                                                                                                                                                                                                                                                                 |
```

Reply:
Two things related things. "Counter" is not defined. I don't think we need "based on a previous contract request" because that is also ambiguous ("based on" is not defined). I would simply state "The provider sends a contract offer." The contract offer by definition is in the context of the contract negotiation but we could also state it explicitly, "The provider sends a contract offer for the referenced contract negotiation"

Reply:
@jimmarino 
> The provider sends a contract offer for the referenced contract negotiation
I would add "[...] contract negotiation process", to have a textual linking to the `processId`. 


@juliapampus 
> The provider sends a (counter) contract offer for a requested dataset, based on a previous contract request.
We might not have a dataset here, since INIT -> OFFERED is possible. So in general: How can the consumer decide, if the offer shall be accepted without a reference to a dataset? A `ContractOfferMessage` just has `processId`, `offer` and `callbackAddress`. Is it possible that this edge may not be effective?